src/Network/Network/ApplicationGateway/RewriteRuleHeaderConfiguration/AzureApplicationGatewayRewriteRuleHeaderConfigurationBase.cs

```diff
@@ -32,6 +32,11 @@ public class AzureApplicationGatewayRewriteRuleHeaderConfigurationBase : Network
             HelpMessage = "Header value to set for the given header name. Header will be deleted if this is omitted")]
         public string HeaderValue { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "[ToDo] Provide help message")]
```

Comment:
should this message be populated before merge ?

Reply:
change has been made.